### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by jaindharmesh52**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

**Comment by stellargo** [comment]
@jaindharmesh52 Hi, so you need to write a commit body, give a link to the issue it fixes :) You could have a look at http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html :)

File: README.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@
 
 =====
 
-`coala <https://coala.io>`_ provides a unified interface for linting and fixing code with a single configuration file, regardless of the programming languages used. You can use coala from within your favorite editor, integrate it with your CI, get the results as JSON, or customize it to your needs with its flexible configuration syntax.
+`coala <https://coala.io>`_ provides a unified interface for linting and fixing the code with a single configuration file, regardless of the programming languages used. You can use coala from your favorite editor, integrate it with your CI, get the results as JSON, or customize it to your needs with its flexible configuration syntax.
```
Comment (adtac) [inline README.rst]: "from within your..." sounds alright to me tbh

**Comment by adtac** [comment]
Also, your commit message requires some work as @stellargo said (see coala.io/commit).

**Comment by adtac** [comment]
unack https://github.com/coala/coala/pull/4795/commits/bbc94958cae036582d17743201094cf3a8320a3f

**Comment by jschwarzwalder** [comment]
unack bbc9495

**Comment by jayvdb** [comment]
Closing abandoned patch without an issue number.

**Pull request closed without merging**